app/metrics.yaml

```diff
@@ -3131,3 +3131,82 @@ perf.awesomebar:
       - fenix-core@mozilla.com
       - gkruglov@mozilla.com
     expires: "2020-10-01"
+
+storage.stats:
+  query_stats_duration:
+    send_in_pings:
+      - metrics
+    type: timing_distribution
+    description: >
+      How long it took to query the device for the StorageStats that contain the
+      file size information. The docs say it may be expensive so we want to
+      ensure it's not too expensive.  This value is only available on Android
+      8+.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/12802
+    data_reviews:
+      - todo
+    notification_emails:
+      - fenix-core@mozilla.com
+      - perf-android-fe@mozilla.com
+      - mcomella@mozilla.com
+    expires: "2020-12-21"
```

Comment:
Since we're already at the beginning of August and 12/21 is going to be a rough time to renew probes, let's bump this to feb 2021

Reply:
What an upgrade 🤩